bin/npm2rpm.js

```diff
@@ -23,6 +23,8 @@ npm2rpm
 .option('-s, --strategy [strategy]', "Strategy to build the npm packages", /^(single|bundle)$/i)
 .option('-r, --release [release]', "RPM's release", 1)
 .option('-t, --template [template]', "RPM .spec template to use")
+.option('-u, --update [update]', "If only wanting to update spec file")
+.option('-6, --use-nodejs6 [useNodejs6]', "If wanting to generate cache tarball for NodeJS 6")
```

Comment:
Note these two options that were added to help with backwards compatibility to NodeJS 6 packages